Src/pangen7.c

```diff
@@ -193,10 +193,12 @@ complete_transition(Succ_List *sl, Guard *g)
 	printf("	:: ");
 	for (w = g; w; w = w->nxt)
 	{	if (w->t->ntyp == CONST
+		&&  w->t->constValKind == VALUE_INT	// TODO PG - verify if it could be float...
 		&&  w->t->val == 1)
 		{	continue;
 		} else if (w->t->ntyp == 'c'
 		&&  w->t->lft->ntyp == CONST
+		&& 	w->t->lft->constValKind == VALUE_INT	// TODO PG - verify if it could be float... and then what?
```

Comment:
Ident. Also, this is something that we could discuss.

Reply:
so far, in all these points, it seems, that float is not an option - as the queues will be implemented, hopefully TODO will be changed to informative info why I do not border with float const here

Reply:
indentetion fixed